app/Http/Controllers/User/AdminOrderInvoiceController.php

```diff
@@ -78,10 +86,39 @@ public function getClientInvoice($id)
                             style='color:white;'> </i>&nbsp;&nbsp;View</a>"
                                     ."   $editAction $action";
                         })
-                         ->rawColumns(['checkbox', 'date', 'invoice_no', 'total', 'paid', 'balance', 'status', 'action'])
+                         ->rawColumns(['checkbox', 'date', 'invoice_no', 'order_no', 'total', 'paid', 'balance', 'status', 'action'])
                         ->make(true);
     }
 
+    public static function getOrderLink($invoice)
```

Comment:
I believe this is getting used in a loop. It will kill performance. Please use joins in the main query

Reply:
fixed